features/google/src/components/dataStructureMiniStory/dataStructureMiniStory.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { PolyChart, FilterChips } from "@polypoly-eu/poly-look";
+
+import "./dataStructureMiniStory.css";
+//This component needs to go to poly-look
```

Comment:
I am not entirely sure if it will go to poly-look or perhaps poly-analysis (?), but just given that the ministories are idententical, apart from the text, I think they should go in a shared package

Reply:
A way to get reminded of that would be to open a mini-Jira ticket. All this rationale can go into there.